packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -87,158 +86,92 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		return <Waiting label="Waiting for Audience data." />;
 	}
 
+	// TODO: Determine if myClientMetaData is necessary
 	const myClientMetadata = audienceData.audienceState.find(
 		(audience) => audience.clientId === audienceData.clientId,
 	)?.client;
 
+	const audienceStateItems = AudienceStateDataFilter(
+		audienceData.audienceState,
+		myClientMetadata,
+	);
+	const audienceHistoryItems = AudienceHistoryDataFilter(audienceData.audienceHistory).reverse();
+
 	return (
-		<Stack
-			styles={{
-				root: {
-					height: "100%",
-				},
-			}}
-		>
-			<StackItem>
-				<h3>Audience members: ({audienceData.audienceState.length})</h3>
-				<MembersView
-					audience={audienceData.audienceState}
-					myClientId={audienceData.clientId}
-					myClientConnection={myClientMetadata}
-				/>
-			</StackItem>
-			<StackItem>
-				<h3>History</h3>
-				<HistoryView history={audienceData.audienceHistory} />
-			</StackItem>
-		</Stack>
+		<>
+			<Divider appearance="brand"> Audience State </Divider>
+			<AudienceStateTable audienceStateItems={audienceStateItems} />
+			<Divider appearance="brand"> Audience History </Divider>
+			<AudienceHistoryTable audienceHistoryItems={audienceHistoryItems} />
+		</>
 	);
 }
 
 /**
- * {@link MembersView} input props.
+ * Filtered audience state data for {@link AudienceStateDataFilter}
  */
-interface MembersViewProps {
-	/**
-	 * The current audience
-	 */
-	audience: AudienceClientMetadata[];
-
-	/**
-	 * My client ID, if the Container is connected.
-	 */
-	myClientId: string | undefined;
-
-	/**
-	 * My client connection data, if the Container is connected.
-	 */
+export interface FilteredAudienceStateData {
+	clientId: string;
+	userId: string;
+	mode: string;
+	scopes: string[];
 	myClientConnection: IClient | undefined;
 }
 
 /**
- * Displays a list of current audience members and their metadata.
+ * Removes unncessary data in audienceData.audienceState
  */
-function MembersView(props: MembersViewProps): React.ReactElement {
-	const { audience, myClientId, myClientConnection } = props;
-
-	const transformedAudience = combineMembersWithMultipleConnections(audience);
-
-	const memberViews: React.ReactElement[] = [];
-	for (const member of transformedAudience.values()) {
-		memberViews.push(
-			<StackItem key={member.userId}>
-				<AudienceMemberView
-					audienceMember={member}
-					myClientId={myClientId}
-					myClientConnection={myClientConnection}
-				/>
-			</StackItem>,
-		);
-	}
-
-	return <Stack>{memberViews}</Stack>;
+function AudienceStateDataFilter(
+	audienceStateData: AudienceClientMetadata[],
+	myClientConnection: IClient | undefined,
+): FilteredAudienceStateData[] {
+	return audienceStateData.map((entry) => {
+		const clientId = entry.clientId;
+		const userId = entry.client.user.id;
+		const mode = entry.client.mode;
+		const scopes = entry.client.scopes;
+
+		return {
+			clientId,
+			userId,
+			mode,
+			scopes,
+			myClientConnection,
+		};
```

Comment:
Nit; since those variables are only used once, "merging" the assignments into the object construction saves us some lines.
```suggestion
		return {
			clientId: entry.clientId,
			userId: entry.client.user.id,
			mode: entry.client.mode,
			scopes: entry.client.scopes,
			myClientConnection,
		};
```

Reply:
Resolved.